Jamulus.pro

```diff
@@ -141,11 +143,22 @@ win32 {
     HEADERS += mac/activity.h
     OBJECTIVE_SOURCES += mac/activity.mm
     CONFIG += x86
-    QMAKE_TARGET_BUNDLE_PREFIX = net.sourceforge.llcon
+    QMAKE_TARGET_BUNDLE_PREFIX = io.jamulus
```

Comment:
We had a discussion on that with @hoffie: shouldn't we use your username here?

Reply:
No, it should be io.jamulus. The ID is for the app, and is transferable.

Reply:
Ok. Thanks